python/GafferScene/ScriptProcedural.py

```diff
@@ -167,4 +169,27 @@ def __allRendered():
 		Gaffer.ValuePlug.setCacheMemoryLimit( 0 )
 		Gaffer.ValuePlug.setCacheMemoryLimit( memoryLimit )
 
+	__errorConnections = {}
+	@classmethod
+	def __ensureErrorConnection( cls, node ) :
+
+		if node in cls.__errorConnections :
+			return
+
+		cls.__errorConnections[node] = node.errorSignal().connect( cls.__error )
+
+	@staticmethod
+	def __error( plug, source, error ) :
+
+		errorContext = "Plug \"%s\"" % source.relativeName( source.ancestor( Gaffer.ScriptNode ) )
+		if "scene:path" in Gaffer.Context.current() :
+			path = "/" + "/".join( [ str( x ) for x in Gaffer.Context.current()["scene:path"] ] )
```

Comment:
Any reason not to use `IECore.SceneInterface.pathToString()` here? I guess to allow non Cortex compliant scene paths in the future?